src/compas_timber/ghpython/components/CT_Show_Elements_by_Type/code.py

```diff
@@ -0,0 +1,19 @@
+# flake8: noqa
+from ghpythonlib.componentbase import executingcomponent as component
+from compas.scene import Scene
+
+
+class ShowElementsByType(component):
+    def RunScript(self, model):
+        beam_scene = Scene()
+        plate_scene = Scene()
+        fastener_scene = Scene()
+
+        for beam in model.beams:
+            beam_scene.add(beam._geometry or beam.blank)
```

Comment:
why not `beam.geometry`?

Reply:
This is so that it doesn't generate the geometry, it uses either the geometry that was already computed, or the blank.